mars/services/task/execution/ray/executor.py

```diff
@@ -193,10 +220,13 @@ def destroy(self):
         self._available_band_resources = None
 
         # For progress
-        self._pre_all_stages_progress = 1
-        self._pre_all_stages_tile_progress = 1
-        self._cur_stage_tile_progress = 1
-        self._cur_stage_output_object_refs = []
+        self._pre_all_stages_progress = None
+        self._pre_all_stages_tile_progress = None
+        self._cur_stage_tile_progress = None
```

Comment:
The progresses should be 1 if the executor is destroyed (when the task is finish)?

Reply:
Fixed.